packages/util-crypto/README.md

```diff
@@ -16,6 +16,20 @@ Functions can be imported as follows:
 import { mnemonicGenerate } from '@polkadot/util-crypto';
 ```
 
-## Utilities
+## Example
 
-[Utilities](SUMMARY.md)
+```js
+import { mnemonicGenerate, mnemonicToBip39, naclKeypairFromSeed, encodeAddress } from '@polkadot/util-crypto';
+
+const mnemonic = mnemonicGenerate()
+console.log('mnemonic:', mnemonic)
+
+const seed = Buffer.from(mnemonicToBip39(mnemonic))
+console.log('seed', seed.toString('hex'))
+
+const keyPair = naclKeypairFromSeed(seed) // using ed25519
```

Comment:
Follow-up - since seed above is not Substrate-compatible, this would be different from anything else generated in the ecosystem, including anything using the JS libs such as the keyring, UIs, extensions.

Reply:
technically would work if I'd provide a minisecret seed instead, right?

Reply:
Yes, 100%. The mini-secret naming aligns just with the rest of substrate, but it, like the legacy format, is 32-bytes in length, so fits in perfectly.